src/shared/lib/MutationTableDownloadDataFetcher.ts

```diff
@@ -81,4 +91,24 @@ export class MutationTableDownloadDataFetcher implements ILazyMobXTableApplicati
             return undefined;
         }
     }
+
+    private async fetchAllfetchDiscreteCNAData()
```

Comment:
Did you mean `fetchAllDiscreteCNAData` ?

Reply:
Updated! Thanks for your time!